Automattic/wp-calypso
# Framework: Refactor and remove FollowsList emitter store

#### Changes proposed in this Pull Request

This PR refactors the `FollowsList` into a simpler, direct approach, and removes the follows list emitter store that we were previously using. At first I thought about using Redux, but then I noticed that the usage is so simple that it is just not worth all the boilerplate.

#### Testing instructions

* Go to `/stats/insights/:site` for a site that has a bunch of visitors and followers.
* Observe the "Comments" and "Followers" sections at the bottom.
* Verify follow / unfollow buttons that appear for each user in those sections still work properly.

#### Notes

ESLint warnings are expected - original files contain them and they are not being introduced in this PR.

This PR fixes #48405.
